src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -35,59 +26,6 @@ public function __construct(Definition $definition, MarkingStoreInterface $marki
         $this->markingStore = $markingStore;
         $this->dispatcher = $dispatcher;
         $this->name = $name;
-
-        // If the marking can contain only one place, we should control the definition
-        if ($markingStore instanceof UniqueTransitionOutputInterface) {
-            foreach ($definition->getTransitions() as $transition) {
-                if (1 < count($transition->getTos())) {
-                    throw new LogicException(sprintf('The marking store (%s) of workflow "%s" can not store many places. But the transition "%s" has too many output (%d). Only one is accepted.', get_class($markingStore), $this->name, $transition->getName(), count($transition->getTos())));
-                }
-            }
-        }
-    }
-
-    /**
-     * Returns the object's Marking.
-     *
-     * @param object $subject A subject
-     *
-     * @return Marking The Marking
-     *
-     * @throws LogicException
-     */
-    public function getMarking($subject)
```

Comment:
Why did you move this method. It makes the diff harder to read. Could you restore the methods order to ease my review?
thanks.